app/models/competition.rb

```diff
@@ -739,11 +741,11 @@ def trainee_delegate_ids
   end
 
   def enable_v2_registrations!
-    update_column :uses_v2_registrations, true
+    update_column :registration_version, 'v2'
```

Comment:
Now with the integer-baser enum column this will probably(?) break. Unless Rails is smart enough to convert between string and symbol.